src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -290,15 +290,15 @@ export default class AircraftCommander {
         const turnDirection = data[0];
         const legLength = data[1];
         const holdFix = data[2];
-        let holdFixLocation = aircraft.position;
+        let holdPosition = aircraft.position;
         let inboundHdg = aircraft.heading;
 
         if (holdFix) {
-            holdFixLocation = this._navigationLibrary.getFixPositionCoordinates(holdFix);
-            inboundHdg = bearingToPoint(aircraft.position, holdFixLocation);
+            holdPosition = this._navigationLibrary.findFixByName(holdFix).position;
```

Comment:
The variable already exists though, and to my knowledge (and what I saw from some googling), it's not possible to destructure _to_ an existing variable. 🤔